src/styles/index.css

```diff
@@ -0,0 +1,128 @@
+@import './reset.css';
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  max-width: 200px;
+  box-sizing: border-box;
+}
+
+.card__content {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+}
+
+.card__image {
+  width: 160px;
+  margin: 16px auto 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__code {
+  margin: 4px 0 16px;
+  font-size: 10px;
+  color: #616070;
+  line-height: 14px;
+}
+
+.card__grades {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__stars-star {
+  width: 16px;
+  height: 16px;
+  background-image: url('../images/star.svg');
+  background-size: auto;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.card__stars-star:last-child {
+  margin-right: 0;
+}
+
+.card__stars--0 .card__stars-star:nth-child(-n + 0) {
+  background-image: url('../images/star-active.svg');
+}
+
+.card__stars--1 .card__stars-star:nth-child(-n + 1) {
+  background-image: url('../images/star-active.svg');
+}
+
+.card__stars--2 .card__stars-star:nth-child(-n + 2) {
+  background-image: url('../images/star-active.svg');
+}
+
+.card__stars--3 .card__stars-star:nth-child(-n + 3) {
+  background-image: url('../images/star-active.svg');
+}
+
+.card__stars--4 .card__stars-star:nth-child(-n + 4) {
+  background-image: url('../images/star-active.svg');
+}
+
+.card__stars--5 .card__stars-star:nth-child(-n + 5) {
+  background-image: url('../images/star-active.svg');
+}
+
+.card__reviews {
+  color: #060b35;
+  font-size: 10px;
+  line-height: 14px;
+  height: 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-block: 24px 16px;
+}
+
+.card__price-title {
+  color: #616070;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price-count {
+  color: #060b35;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  padding: 12px;
+  border: 1px solid #00acdc;
+  font-weight: bold;
+  color: white;
+  font-size: 14px;
+  line-height: 16px;
+  background: #00acdc;
+  border-radius: 5px;
+  margin: 0 1px;
```

Comment:
The `margin` property is not necessary here if you already set the `padding` property in the `.card__button` class. It's better to remove it to avoid confusion.